internal/dashboard/business/oceanbase/obcluster.go

```diff
@@ -478,64 +492,125 @@ func generateOBClusterInstance(param *param.CreateOBClusterParam) *v1alpha1.OBCl
 	return obcluster
 }
 
-func CreateOBCluster(ctx context.Context, param *param.CreateOBClusterParam) error {
+func JudgeResourceEnoughForOBCluster(ctx context.Context, obcluster *v1alpha1.OBCluster) error {
```

Comment:
consider node selector and affinity?

Reply:
It's difficult to analyze affinity in, this function is merely a loose or an approximate way to judge whether all remain resource is enough for new coming oceanbase cluster. It prevents some mistakes.

Reply:
I will suggest to delete the validation then, if you can't do the validation precisely by design, then it's better not to do it.

Reply:
Let's discuss about it tomorrow ~